src/schema/v2/homeView/sections/QuickLinks.ts

```diff
@@ -32,6 +37,64 @@ function getDisplayableQuickLinks(context: ResolverContext) {
   })
 }
 
+export const QUICK_LINKS_V2: Array<NavigationPill> = [
+  {
+    title: "Discover Daily",
+    href: "/infinite-discovery",
+    ownerType: OwnerType.infiniteDiscovery,
+    icon: "ImageSetIcon",
+    minimumEigenVersion: { major: 8, minor: 67, patch: 0 }, // same constraint as InfiniteDiscovery section
+  },
```

Comment:
@iskounen Since we are now going to link to Discover Daily from the quick links, I want to double-check this version constraint.

Reply:
this is a nice addition but because we were careful in Eigen as well, we should be safe also without this check

Reply:
If you're looking to remove that minimum version, I think it's OK because enough users are on a newer version than that.

Reply:
Sounds like it's fine either way, so I'm just gonna merge as-is. Thanks!